packages/ui-components/src/components/search.tsx

```diff
@@ -185,18 +184,16 @@ export const FilterBox = (props: IFilterBoxProps): JSX.Element => {
       target.value
     );
   };
-
   return (
-    <InputGroup
-      className="jp-FilterBox"
-      inputRef={props.inputRef}
-      type="text"
-      disabled={props.disabled}
-      rightIcon="ui-components:search"
-      placeholder={props.placeholder}
-      onChange={handleChange}
+    <Search
+      ref={props.inputRef}
       value={filter}
-    />
+      onChange={handleChange}
+      placeholder={props.placeholder}
+      disabled={props.disabled}
+    >
+        <searchIcon.react slot='end' />
```

Comment:
We should not wrap the svg within another DOM element:

```suggestion
        <searchIcon.react slot='end' tag={null}/>
```